packages/app-admin/src/components/AppInstaller/AppInstaller.tsx

```diff
@@ -40,12 +41,23 @@ export const AppInstaller: React.FC = ({ children }) => {
         loading,
         installers,
         installer,
+        isFirstInstall,
         showNextInstaller,
         showLogin,
         onUser,
         skippingVersions
     } = useInstaller({ isInstalled: isInstallerCompleted() });
 
+    const openWelcomeScreen = () => {
+        if (!isRootTenant) {
+            return;
+        }
+        if (!isFirstInstall) {
+            window.location.href = `https://site.webiny.com/thank-you/upgrade?version=${wbyVersion}&returnUrl=${window.location.href}`;
```

Comment:
There's a big chance that this will not redirect soon enough, and the second `window.location.href` on line `58` will kick in. Just add a `return` after this to stop the execution.

I tried 2 subsequent `window.location.href` in the browser dev tools, and the last one wins.

Second question: is `site.webiny.com` a permanent domain we want to have here?

Reply:
Thanks Pavel, good catch, I've updated this.

Yes, site.webiny.com is a domain we own & use for various lead capturing information.